dimod/binary_quadratic_model.py

```diff
@@ -123,17 +130,48 @@ class for the K4 complete graph, where the nodes have biases
     BINARY = Vartype.BINARY
 
     @vartype_argument('vartype')
-    def __init__(self, linear, quadratic, offset, vartype):
+    def __init__(self, linear, quadratic, offset, vartype, **kwargs):
         self.linear = {}
         self.quadratic = {}
         self.adj = {}
         self.offset = offset  # we are agnostic to type, though generally should behave like a number
         self.vartype = vartype
+        self.info = kwargs  # dump any additional kwargs into info
 
         # add linear, quadratic
         self.add_variables_from(linear)
         self.add_interactions_from(quadratic)
 
+    @classmethod
+    def empty(cls, vartype):
+        """Creates an empty BinaryQuadraticModel.
+
+        Equivalent to
+
+        .. code-block:: python
+
+            BinaryQuadraticModel({}, {}, 0.0, vartype)
+
+        Args:
+            vartype (:class:`.Vartype`/str/set):
+                The variable type desired for the binary quadratic model. Accepted input values:
```

Comment:
Variable type for the binary quadratic model.

Reply:
I copied this from you! :laughing:

Reply:
I changed it in both places.